### ISSUE TITLE
Error when using hint example number (Ksovo)

### ISSUE THREAD

**Issue by AliYusuf95** [hbb20/CountryCodePickerProject#326]
I used `ccp_hintExampleNumber="true"` option on my layout and everything is working fine until I change the country to `Ksovo` while the input already filled with number `12312312`.
```java
Attempt to invoke virtual method 'java.lang.String java.lang.String.substring(int)' on a null object reference
    java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.String.substring(int)' on a null object reference
        at com.hbb20.CountryCodePicker.updateHint(CountryCodePicker.java:864)
        at com.hbb20.CountryCodePicker.setSelectedCountry(CountryCodePicker.java:817)
        at com.hbb20.CountryCodePicker.onUserTappedCountry(CountryCodePicker.java:2261)
        at com.hbb20.CountryCodeAdapter$4.onClick(CountryCodeAdapter.java:180)
        at android.view.View.performClick(View.java:5198)
        at android.view.View$PerformClick.run(View.java:21147)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

It seems in `updateHint` function the `formattedNumber` became null from the `PhoneNumberUtils.formatNumber` then it's try to call `substring` on `null` value

**Comment by dmitriy1morozov** [comment]
Checked the error. Try to select mentioned Kosovo country.

I personally found identically the same bug with Vatican(VA +379) country. I always get nullPointer exception

**Comment by hbb20** [comment]
Thank you for reporting. We just released new version `v2.3.0` with this fix. Please check and let us know if it does not work as expected.
